tools/docker/README.md

```diff
@@ -7,10 +7,12 @@ Asterinas development Docker images are provided to facilitate developing and te
 To build a Docker image for Asterinas and test it on your local machine, navigate to the root directory of the Asterinas source code tree and execute the following command:
 
 ```bash
+# Generate Dockerfile
+python3 ./tools/docker/gen_dockerfile.py
 docker buildx build \
-    -f tools/docker/Dockerfile.ubuntu22.04 \
-    --build-arg ASTER_RUST_VERSION=$RUST_VERSION \
-    -t asterinas/asterinas:$ASTER_VERSION \
+    -f tools/docker/Dockerfile \
```

Comment:
This looks weird. A CLI tool normally outputs the file in the current working directory, not the directory of the script.